03_how_ln_works.asciidoc

```diff
@@ -609,159 +621,117 @@ These fees are paid to the miner who mines that particular block.
-The payer and payee must cooperate, i.e. in some fashion one could say "synchronize" , during the payment. 
-If the payee's node is down the payer cannot make the payment.
-
-==== Satoshis vs. Milli-Satoshis
-On Bitcoin the smallest amount is a "satoshi" which cannot be divided any further.
-Lightning can be more flexible. 
-Lightning nodes can optionally be configured to work with "milli-satoshis". 
-So, within Lightning even smaller amounts can be exchanged. 
-When settled on-chain, it must be settled in "satoshis".
-
-==== Blockchain Space as Scarcity vs. Channel Capacity as Scarcity
-Bitcoin and Lightning have different scarce resources. 
-The scarce resource for operating on the Bitcoin protocol is the space in the blockchain. 
-The scarce resource for operating on Lightning is the channel capacity, or in other words the liquidity.
+When a Bitcoin user sends funds to a destination address he does not need to know anything about the recipient. The recipient could be might be offline or online, and no interaction between sender and recipient is needed. The interaction is between sender and the Bitcoin blockchain. Receiving bitcoin on the Bitcoin blockchain is a _passive_ and _asynchronous_ activity that does not require any interaction by the recipient, or for the recipient to be online at any time. Bitcoin addresses can even be generated offline and are never "registered" with the Bitcoin network. Only spending bitcoin requires interaction.
+
+In Lightning, the recipient must be "online" during the transaction.
+The recipient must run a node or have someone that runs a node on their behalf (a third-party custodian). To be precise, both nodes, the sender's and the recipient's must be online at the time of payment and must coordinate. Receiving a Lightning payment is an _active_ and _synchronous_ activity between sender and recipient, without the participation of most of the Lightning network or the Bitcoin network (except for the intermediary routing nodes, if any).
+
+The synchronous and always-online nature of the Lightning network is probably the biggest difference in the user experience and often confounds users who are accustomed to Bitcoin.
+
+==== Satoshis vs. milli-satoshis
+
+On Bitcoin the smallest amount is a _satoshi_ which cannot be divided any further. Lightning is a bit more flexible, and Lightning nodes work with _milli-satoshis_ (thousandths of a satoshi). This allows tiny payments to be sent via Lightning. A single milli-satoshi payment can be comfortably sent across a payment channel, which based on current bitcoin value would be the equivalent of a hundredth thousandth of a US cent (or $0.00000009 USD). Less than a micropayment, this has been characterized as a _nanopayment_. Routing across multiple channels, such an amount would likely be overwhelmed by fees, but some nodes advertize zero-fee routing, so even that is currently possible.
```

Comment:
remove comparison to USD. 1 BTC is always 100'000'000 satoshi. exchange rates to fiat currencies are not stable at this point and why USD? we could also use the EURO or GOLD or the venezuelian bolivar (: